contracts/api3-server-v1/proxies/interfaces/IApi3CrossRateReaderProxyV1.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "../../../interfaces/IApi3ReaderProxy.sol";
+
+interface IApi3CrossRateReaderProxyV1 is IApi3ReaderProxy {
+    enum CalculationType {
+        Divide1By2,
+        Divide2By1,
+        Multiply
```

Comment:
Is there an existing need?

Reply:
I was able to only find cases where we multiply the proxies tbh 🤔

Reply:
I think there was also a case when a dApp used division, but I can't remember which one it was. It probably wasn't for our feeds though.

Reply:
https://github.com/api3dao/contracts/pull/472/commits/5512753d18a40b90ee9ac520bfeb5b3efe295862